src/xbot/common/subsystems/BasePoseSubsystem.java

```diff
@@ -47,8 +47,7 @@
     
     private BooleanProperty rioRotated;
     
-    @Inject
-    public PoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
+    public BasePoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
```

Comment:
Does it make sense for this to be a TelemetrySource now since the drive part can now be a poll?